src/core/qwindowkit_windows.h

```diff
@@ -134,6 +137,33 @@ namespace QWK {
     inline bool WindowsRegistryKey::isValid() const {
         return m_key != nullptr;
     }
+
+    template<>
+    inline const std::optional<DWORD> WindowsRegistryKey::value(QStringView subKey) const {
+        auto dv = dwordValue(subKey);
```

Comment:
I was meant to make this "dv" const, but never mind, const or not doesn't really matter.